en/services/component_information.md

```diff
@@ -179,18 +180,75 @@ Specifically, the following information is provided:
 
 A GCS can provide a UI for testing outputs based on the configured output functions, by iterating over all output channels and collecting the configured actuator output functions, and then utilizing the `MAV_CMD_ACTUATOR_TEST` command.
 
+## Translation
 
-## Open Issues
+At high-level, metadata translation works as following:
+
+- The metadata provider sets the `translationUri` in [general metadata file](#COMP_METADATA_TYPE_GENERAL) for each metadata type that supports translation.
+  There is no CRC for the resource pointed to by this URI, as translations might change independently of metadata (for example, adding or changing translations for a particular language, or adding translations for a new language).
+- The `translationUri` URL points to a summary JSON file that contains links to the separate files that contain each translation of the particular metadata type.
+  The translation summary JSON file also contains modification timestamps for each linked translation file so that a GCS can determine whether the file has been updated.
+  The translation files are a (compressed) file in [TS file format](https://doc.qt.io/qt-6/linguist-ts-file-format.html).
+- A client (GCS) downloads the summary file then uses its contained to locate and download the translation file(s) it is interested in.
+- The client can then apply the downloaded translations to the metadata json file(s) (which contains annotations for which tags to translate) using the translation TS.
+
+### Caching
+
+The following caching strategy is recommended for clients:
+
+- locally cache the downloaded files for 3 days before re-checking
+- after 3 days try to download the summary again, and if no internet connection, keep using the cached version
+- the translation files can either be downloaded whenever the summary is downloaded or by checking the modification timestamp in the summary
```

Comment:
```suggestion
The following caching strategy is recommended for clients:

- Locally cache the downloaded translation files.
  These should be used until successfully replaced with a newer version.
- After 3 days attempt to download the summary JSON file again.
- Translation files can either be downloaded whenever the summary is downloaded or only when needed (because a modification timestamp has changed in the summary).
```